docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,313 @@
+
+        for arg in argv do
+            printfn "%s" <| getJsonNetJson arg
+
+        0 // return an integer exit code
+    ```
+7. Enter `dotnet restore` and `dotnet build` into the command line.  These should succeed.
+8. Enter `dotnet run Hello World` into the command line.  You should see results like this:
+
+```
+Nice command line arguments!  Here's what JSON.NET has to say about them:
+
+I used to be Hello but now I'm ""Hello""!
+I used to be World but now I'm ""World""!
+```
+
+Your `project.json` file should look like this:
+
+```json
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": true,
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Program.fs"
+      ]
+    }
+  },
+  "tools": {
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-preview2-*",
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win81",
+        "netstandard1.3"
+      ]
+    }
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        },
+        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+        "Library":{
+          "version":"1.0.0",
+          "target": "project"
+        }
+      }
+    }
+  }
+}
+```
+
+And your `Program.fs` file should look like this:
+
+```fsharp
+open System
+open Library
+
+[<EntryPoint>]
+let main argv = 
+    printfn "Nice command line arguments!.  Here's what JSON.NET has to say about them:"
+
+    for arg in argv do
+        printfn "%s" <| getJsonNetJson arg
+
+    0 // return an integer exit code
+```
+
+### Testing the Class Library with xUnit.net
+
+1. Create a `TestLibrary` folder under `NETCoreFS/test`.
+2. In the command line, execute `dotnet new -l F#` in `FSNetCore/src/Tests`.
+3. Remove the `NuGet.Config`.
+4. Rename `Program.fs` to `Tests.fs`.
+5. Open the `project.json` file.
+6. Remove the `emitEntryPoint` entry under `buildOptions`.
+7. Under `buildOptions/compile/includeFiles`, replace `Program.fs` with `Tests.fs`.
+8. Remove the global `dependencies` section. 
+9. Under `frameworks/netcoreapp1.0/`, remove the `imports` section.
+10. Under `frameworks/netcoreapp1.0/dependencies`, add the following after `Microsoft.NETCore.App`:
+    ```json
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+    "xunit":"2.2.0-beta2-build3300",
+    "dotnet-test-xunit":"2.2.0-preview2-build1029",
+    "Library":{
+      "version": "1.0.0",
+      "target": "project"
+    },
+    ```
+11. After the `frameworks` section, add `"testRunner":"xunit"`.  Note that you can add this section anywhere in the `project.json` file.
+12. In `test.fs`, paste the following code:
+    ```fs
+    module Test
+
+    open Xunit
+    open Library
+
+    [<Fact>]    
+    let ``Library converts "Banana" correctly``() =
+        let expected = @"I used to be hey but now I'm ""hey""!"
```

Comment:
use `"""` (triple `"`) to avoid `@` and double `"` inside string, so expected string is exactly the same

```fsharp
"""I used to be hey but now I'm "hey"!"""
```